tests/transforms/test_mitigate.py

```diff
@@ -326,3 +332,53 @@ def mitigated_circuit(w1, w2):
         g = qml.grad(mitigated_circuit)(w1, w2)
         for g_ in g:
             assert not np.allclose(g_, 0)
+
+
+class TestFoldGlobal:
+    """Testing ``qml.transforms.fold_global"""
+
+    def test_constant_result(self):
+        """Ensuring that the circuit always yields the same result upon folding."""
+
+        dev = qml.device("default.qubit", wires=5)
+
+        # Select template to use within circuit and generate parameters
+        n_layers = 2
+        n_wires = 3
+        template = qml.SimplifiedTwoDesign
+        weights_shape = template.shape(n_layers, n_wires)
+        w1, w2 = [np.arange(np.prod(s)).reshape(s) for s in weights_shape]
+
+        # This circuit itself produces the identity by construction
+        with qml.tape.QuantumTape() as circuit:
+            template(w1, w2, wires=range(n_wires))
+            qml.adjoint(template(w1, w2, wires=range(n_wires)))
+            qml.expval(qml.PauliZ(0))
+
+        folded_qnodes = [fold_global(circuit, scale_factor=lambda_) for lambda_ in np.arange(1, 10)]
+        res = [qml.execute([folded], dev, None) for folded in folded_qnodes]
+        assert np.allclose(res, 1)
+
+
+class TestPolyfit:
+    """Testing that polyfit correctly fits polynomials"""
+
+    @pytest.mark.all_interfaces
+    def test_polyfit(self):
+        """Testing the poly_extrapolator function in"""
+        import jax.numpy as jnp
```

Comment:
I saw it explained somewhere but cannot find it anymore. How best to import other interfaces for tests again? Importing it inside the file leads to errors